src/kakarot/instructions/memory_operations.cairo

```diff
@@ -100,7 +101,14 @@ namespace MemoryOperations {
         let offset = popped[1];
         let value = popped[0];
 
-        let memory: model.Memory* = Memory.store(self=ctx.memory, element=value, offset=offset.low);
+        
+        let (bytes_array_len, bytes_array) = Helpers.uint256_to_bytes_array(value);
+
+        with_attr error_message("Uint256 represented as an array of bytes should be of length 32") {
+            assert bytes_array_len = 32;
+        }
```

Comment:
this is more a test of the helpers than something required here right?

Reply:
yeah fair